Community/News/SCMS-is-now-a-chaoss-metric.md

```diff
@@ -1,54 +1,67 @@
-This past January, our friends over at CHAOSS Bitergia announced the newest round of community metrics to go into their community analytics system, GrimoireLab, and nestled in those new metric updates was a type of metric that has not yet been featured before in CHAOSS... Our Social Currency Metrics System (SCMS)!
+### By Samantha Venia Logan
 
-Then on May 4th, we were told that we had been accepted alongside side software developer Ria Gupta for the Google Summer of Code, to write the SCMS directly into GrimoireLab as a metric! 
+_originally published on May 7th 2020 at [https://www.sociallyconstructed.online/blogs/scms-is-an-official-chaoss-metric](https://www.sociallyconstructed.online/blogs/scms-is-an-official-chaoss-metric)_
+
+This past January, our friends over at [CHAOSS](https://chaoss.github.io/grimoirelab/) announced a new release of health metrics, and nestled in those new metric updates was a type of metric that has not yet been featured before in CHAOSS... [Our Social Currency Metrics System (SCMS)!](https://chaoss.community/metric-social-currency-metric-system/)
+
+Then on May 4th, we were told that we had been accepted alongside side software developer [Ria Gupta](https://medium.com/@guptaria/my-journey-of-gsoc20-begins-268ff97c2954) for the Google Summer of Code, to write the SCMS _directly_ into CHAOSS community analytics system -- [GrimoireLab](https://chaoss.github.io/grimoirelab/) -- as a metric!
+
+![GrimoireLab](https://www.sociallyconstructed.online/uploads/2/5/6/3/25632458/grimoirelab-logo_orig.jpg)
+
+_**In this blog**_ we'll explain why we’ve been added to GrimoireLab, why we think it matters for you, and most importantly what the next steps are in this exciting new process, so you can implement the system, for free, starting today!
+
+## WHY THE SCMS WAS ADDED,​ AND WHERE IT WILL TAKE YOU
```

Comment:
This is a heder and it seems to be showing as such down below so it should be normal in the paragraph above.